lib/railway_ipc/rabbitmq/adapter.rb

```diff
@@ -23,6 +23,7 @@ def initialize(amqp_url: ENV["RAILWAY_RABBITMQ_CONNECTION_URL"], exchange_name:,
                                     user: settings[:user],
                                     pass: settings[:pass],
                                     port: settings[:port],
+                                    vhost: settings[:vhost],
```

Comment:
This shouldn't break anything currently using the gem, because it will already have `ENV["RAILWAY_RABBITMQ_CONNECTION_URL"]` set, and this vhost is pulled from that env variable, right?

Reply:
yep, i will test against ironboard tho

Reply:
Mostly just wondering if we're vulnerable to a situation where we're replacing Bunny's default `vhost` argument with `nil`?

Reply:
it defaults to `/`

Reply:
I see—so should we be worried about cases where 

```ruby
AMQ::Settings.parse_amqp_url(amqp_url)[:vhost]
#=> nil
```

where the `/` default will be overwritten by the explicit passing of `nil` as `vhost`?

Reply:
That is, maybe we wanna do something like
```suggestion
                                    vhost: settings.fetch(:vhost, '/'),
```

to ensure we don't overwrite the would-be default?

Reply:
Yea, let's do this.

Reply:
I updated this a bit differently. Our version of the bunny gem might provide an empty value here. Meaning `settings.fetch(:vhost)` could return `nil` and never default to `'/'`